.github/workflows/ci-containers.yaml

```
@@ -101,8 +95,7 @@ jobs:
 
       - name: Apply local service using image from this run
         run: |
-          sed -i 's|image: calyptia/vivo-service|image: ghcr.io/calyptia/vivo/service@${{ needs.build-container-images.outputs.service-digest }}|g' ./vivo-deployment.yaml
-          sed -i 's|image: calyptia/vivo-frontend|image: ghcr.io/calyptia/vivo/frontend@${{ needs.build-container-images.outputs.frontend-digest }}|g' ./vivo-deployment.yaml
+          sed -i 's|image: calyptia/vivo|image: ghcr.io/calyptia/vivo/vivo@${{ needs.build-container-images.outputs.digest }}|g' ./vivo-deployment.yaml
```

Probably keep it down to a simpler image:
```suggestion
          sed -i 's|image: calyptia/vivo|image: ghcr.io/calyptia/vivo@${{ needs.build-container-images.outputs.digest }}|g' ./vivo-deployment.yaml
```suggestion

---

And after using `yq` I think it's a much better option here - sed is too brittle and messes up formatting. I'd suggest we use it: https://github.com/calyptia/core-images/blob/971a7af9bc3b7640f921e8b87134e15be5b208d0/on-prem/cloud/build-package.sh#L40